DocumentedExamples/Meridional_heat_transport.ipynb

```diff
@@ -5,15 +5,19 @@
    "metadata": {},
```

Comment:
<p>As described at https://github.com/COSIMA/access-om2/issues/139#issuecomment-639278547, for full accuracy we should also be including the frazil component of the surface heat flux via the diagnostic <code>frazil_3d_int_z</code>. If this diagnostic is not available for this run, please make a note of it.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>